src/commands/accounts/remove.ts

```diff
@@ -27,7 +27,8 @@ export default class AccountsRemove extends BaseCommand<typeof AccountsRemove> {
    * @returns Empty promise
    */
   public async run(): Promise<void> {
-    const accountsDomain = await Accounts.init(this.flags['keyring-backend'] as BackendType, { filesPath: this.flags['keyring-path'] });
+    const config = await Config.init();
+    const accountsDomain = await Accounts.init(this.flags['keyring-backend'] || config.keyringBackend, { filesPath: this.flags['keyring-path'] });
```

Comment:
Having this `flags || config` logic repeat everywhere is error-prone.

Instead, please create another method in the `Accounts` called `initFromFlags<T>(flags: Flags<T>, config: Config)` and move the logic for the keyring backend and path there. Also, let's allow the user to override the keyring-path from the config.